tests/Validators/StockLevelValidatorTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Tests\Validators;
+
+use App\Validators\StockLevelValidator;
+use Tests\TestCase;
+
+class StockLevelValidatorTest extends TestCase
+{
+    public function testValidateStockLevelSuccess()
+    {
+        $stockLevel = '1234';
+        $result = StockLevelValidator::validateStockLevel($stockLevel);
+        $this->assertEquals('1234', $result);
+    }
+
+    public function testValidateStockLevelFailure()
+    {
+        $stockLevel = 'ABCD';
+        $this->expectException(\Exception::class);     
```

Comment:
This should be more specific for our case. This just expects any exception but we are expecting our specific custom exception.